src/Web/Avalonia.Blazor/AssetLoader.cs

```diff
@@ -0,0 +1,398 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Avalonia.Platform;
+using Avalonia.Utilities;
+
+namespace Avalonia.Blazor
```

Comment:
Looks like a copy-paste, consider including it via `<Compile Include="../...`

Reply:
done.